### DISCUSSION TITLE
missing coordinate reference system

### DISCUSSION THREAD

**Discussion by wiesehahn** [r-lidar/lidR#465] [Q&A] [answered]
I have LAZ files with additional xml files where the coordinate reference system seems to be stored. 
Reading the LAZ files seems to work as expected, and checking the quality with `las_check(las)` results in `- Checking coordinate reference sytem... ✓`. However, `print(las)` results in `coord. ref.  : NA`.

Shouldn't the crs be read correctly in the first step? And if it is not shouldn't the quality check give a hint?

**Comment by Jean-Romain** [comment, marked as answer]
Well, the CRS is expected to be stored in the LAS/LAZ file not in an additional file. In your case the LAZ files does not contain any CRS. It is not invalid to do not have a CRS and consequently `las_check()` does not report any problem (sometime it reports trouble if the EPSG code correspond to nothing known). Should it really report something? I guess it could be convenient but not mandatory in my opinion.
